tic-tac-toe.js

```diff
@@ -1,7 +1,76 @@
-function TicTacToe() {
-
+function TicTacToe(player1, player2) {
+  this.board = [[1,2,3],[4,5,6],[7,8,9]];
+  this.playerOne = { key: "X",
+                    name: player1};
+  this.playerTwo = { key: "O",
+                    name: player2};
+  this.currentPlayer = this.playerOne;
+  this.gameOver = false;
+  this.turns = 0;
 }
 
-TicTacToe.prototype = {
-  
-}
+TicTacToe.prototype.play = function(clicked){
+
+  if (this.gameOver === false) {
+    this.turn(clicked);
+    this.checkWinner();
+    if (this.gameOver === true){
+      $("#alert").text(this.currentPlayer.name + " is the winner!");
+    } else if (this.turns === 9){
+      this.gameOver = true;
+      $("#alert").text("The cat got the game!");
+      $("<img src=\"http:\/\/www.cats.org.uk/uploads/images/pages/Feliway_that_purple_cat_228.jpg\">").insertAfter("#alert");
+    } else {
+      this.togglePlayer();
+    }
+  }
+};
+
+TicTacToe.prototype.turn = function(clicked) {
+  var row = $(".row");
+  for (var i=0; i < this.board.length; i++) {
+    var squares = $(row[i]).children();
+    var content = this.board[i];
+    for (var i2=0; i2 < content.length; i2++) {
```

Comment:
Naming conventions that we use for loops within loops are to use the `i` for the outer incrementor and `j` for the inner one